pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1377,6 +1391,9 @@ func testClusterDeployment() *hivev1.ClusterDeployment {
 		},
 	}
 
+	cd.Labels = make(map[string]string)
```

Comment:
To make this more resilient to future changes in `testEmptyClusterDeployment`, move the creation of the `Labels` map to `testEmptyClusterDeployment`.